### PR TITLE
Remove deprecated options from Ocata Trove

### PR DESCRIPTION
**Pull request by stephenbma**
Removed the parameter verbose.

### CODE REVIEW

File: chef/data_bags/crowbar/template-trove.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "trove": {
       "crowbar-revision": 1,
       "crowbar-applied": false,
-      "schema-revision": 101,
+      "schema-revision": 201,
```
Comment (nicolasbock) [inline chef/data_bags/crowbar/template-trove.json]: We usually start migrations with `200`.

Comment (stephenbma) [inline chef/data_bags/crowbar/template-trove.json]: Will revise.  Thank you.

**Comment by stephenbma** [comment]
These deprecation warnings still need to be addressed:
`2017-08-16 15:45:16.713 19936 WARNING keystonemiddleware.auth_token [-] AuthToken middleware is set with keystone_authtoken.service_token_roles_required set to False. This is backwards compatible but deprecated behaviour. Please set this to True.
2017-08-16 15:45:16.746 19936 INFO oslo_service.service [-] Starting 2 workers
2017-08-16 15:45:16.785 20054 INFO eventlet.wsgi [-] (20054) wsgi starting up on http://0.0.0.0:8779
2017-08-16 15:45:16.791 19936 WARNING oslo_config.cfg [-] Option "rabbit_password" from group "oslo_messaging_rabbit" is deprecated for removal.  Its value may be silently ignored in the future.`

File: chef/data_bags/crowbar/template-trove.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
...
       "element_states": {
         "trove-server": [ "readying", "ready", "applying" ]
       },
```
Comment (stephenbma) [inline chef/data_bags/crowbar/template-trove.json]: also remove line #7 in this file:
    verbose: true

**Comment by stephenbma** [comment]
Oops, closed this PR by mistake

**Pull request closed without merging**